app/models/user.rb

```diff
@@ -1,8 +1,9 @@
 class User < ActiveRecord::Base
-  has_many :games
-  has_many :pieces
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
+
+  has_many :games
+  has_many :pieces
```

Comment:
You know, I thought the same thing - this is a remnant from the initial setup, I just moved its location in the code. It's echoed in `piece.rb`
```
class Piece < ActiveRecord::Base
  belongs_to :user
  belongs_to :game
```